core/transport/src/main/proto/proto/shared/acl.proto

```diff
@@ -50,3 +57,21 @@ message AccessControlList {
     optional bool isDefault = 7;
     optional bool isEmpty = 8;
 }
+
+// next available id: 10
+message NewAccessControlList {
+    optional string owningUser = 1;
+    optional string owningGroup = 2;
+    optional uint32 mode = 3;
+    repeated ExtendedAclAction namedActions = 4;
+    repeated ExtendedAclAction groupedActions = 5;
+    optional grpc.Bits maskActions = 6;
```

Comment:
where is 7?

Reply:
also have you considered sharing the same message between v1 and v2 and use a couple of "oneof"s for unique fields?